module-bsp/board/rt1051/bsp/eink/ED028TC1.cpp

```diff
@@ -680,26 +681,33 @@ static void s_EinkSetInitialConfig()
     tmpbuf[3] = 0x01;                  // 0x01
     tmpbuf[4] = 0xE0;                  // 0xE0
     if (BSP_EinkWriteData(tmpbuf, 5, SPI_AUTOMATIC_CS) != 0) {
-        return;
+        return EinkDMAErr;
     }
 
     tmpbuf[0] = EinkVCM_DCSetting; // 0x82
     tmpbuf[1] = 0x30;
     if (BSP_EinkWriteData(tmpbuf, 2, SPI_AUTOMATIC_CS) != 0) {
-        return;
+        return EinkDMAErr;
     }
+    return EinkOK;
 }
 
 EinkStatus_e EinkResetAndInitialize()
 {
     BSP_EinkLogicPowerOn();
 
     // Initialize the synchronization resources, SPI and GPIOs for the Eink BSP
-    BSP_EinkInit(NULL);
+    if (BSP_EinkInit(NULL) != 0) {
```

Comment:
What about `nullptr`? :)

Reply:
Parameter was completely unused so I removed it :)